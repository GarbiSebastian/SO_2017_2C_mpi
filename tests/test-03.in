member palabras
member que
member no
member estan
q
