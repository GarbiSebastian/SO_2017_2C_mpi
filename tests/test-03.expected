Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> El string <palabras> no está
> El string <que> no está
> El string <no> no está
> El string <estan> no está
> Finalizamos ejecución
